.vib/haproxy-intel/vib-verify.json

```diff
@@ -0,0 +1,49 @@
+{
+  "phases": {
+    "package": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/haproxy-intel"
+        }
+      },
+      "actions": [
+        {
+          "action_id": "helm-package"
+        },
+        {
+          "action_id": "helm-lint"
+        }
+      ]
+    },
+    "verify": {
+      "context": {
```

Comment:
We should define some basic `runtime_parameters` that we need, even if they are the default values in the chart. These values usually are setting the Service to LoadBalancer and the exposed port to 80/443.

There are some assets that currently do not follow this approach, but defining explicit `runtime_parameters` gives a much more clear idea of what are the values used/required for testing.